modules/ossm-rn-new-features.adoc

```diff
@@ -23,118 +24,16 @@ Result – If changed, describe the current user experience
 |Component |Version
```

Comment:
We're going to need a new ProductVersion variable. In the preview, `Component versions included in {ProductName} version {ProductVersion}` resolves to 1.1.8.

Reply:
Good catch!